data/project/ember/beta.md

```diff
@@ -5,14 +5,14 @@ filter:
   - /ember\./
   - /ember-template-compiler/
 repo: emberjs/ember.js
-initialVersion: 3.15.0
-lastRelease: 3.16.0-beta.1
-futureVersion: 3.16.0-beta.1
-finalVersion: 3.16.0
+initialVersion: 3.16.0
+lastRelease: 3.17.0-beta.5
+futureVersion: 3.17.0-beta.5
+finalVersion: 3.17.0
 channel: beta
-cycleEstimatedFinishDate: 2020-01-20
-date: 2019-12-10
-nextDate: 2019-12-16
+cycleEstimatedFinishDate: 2020-03-25
```

Comment:
The release dates are actually calculated from the original previous date, regardless of delays in the release (like happened with 3.16).

```suggestion
cycleEstimatedFinishDate: 2020-03-02
```